fem/fespace.hpp

```diff
@@ -320,12 +408,36 @@ class FiniteElementSpace
    bool Conforming() const { return mesh->Conforming(); }
    bool Nonconforming() const { return mesh->Nonconforming(); }
 
+   /// Sets the order of the i'th finite element.
+   /** By default, all elements are assumed to be of fec->GetOrder(). Once
+       SetElementOrder is called, the space becomes a variable order space. */
+   void SetElementOrder(int i, int p);
+
+   /// Returns the order of the i'th finite element.
+   /** @note This is the 'order' of the FiniteElementCollection. The actual
```

Comment:
Would it be clearer to say the `order` of the element rather than `FiniteElementCollection`? The collection can have multiple (variable) orders.

Reply:
This refers to the `order` parameter the user constructed the `FiniteElementCollection` with. The point is that this may not be the actual polynomial degree (there is an unfortunate quirk in the naming of the RT collections).

Reply:
In the variable order case, this returns `elem_order[i]` which is the order for the element. Isn't this different from the `FiniteElementCollection` order?

Reply:
There are two things: `order` as passed at the very beginning to `FiniteElementCollection`, this is the same kind of order as `elem_order[i]`, and then there's the real polynomial degree as returned by `FiniteElement::GetOrder()`. For example, `RT_FiniteElementCollection` of order 0 contains `FiniteElements` whose `GetOrder` returns 1.